### PR TITLE
chore(release): 4.0.0

### LINKED ISSUE #54: Extract imports without path quotes

### PR DESCRIPTION
**Pull request by alexander-akait**
fixes #54

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,10 +1,11 @@
 {
   "name": "icss-utils",
-  "version": "3.0.1",
+  "version": "4.0.0",
   "description": "ICSS utils for postcss ast",
   "main": "lib/index.js",
   "files": [
-    "lib"
+    "lib",
+    "src"
```
Comment (alexander-akait) [inline package.json]: **NEED REMOVE** before release (require for tests and migration other plugins)

File: package.json [hunk 2]
```diff
@@ -1,10 +1,11 @@
 {
   "name": "icss-utils",
-  "version": "3.0.1",
+  "version": "4.0.0",
   "description": "ICSS utils for postcss ast",
-  "main": "lib/index.js",
+  "main": "src/index.js",
```
Comment (alexander-akait) [inline package.json]: **NEED CHANGE** before release (require for tests and migration other plugins)

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,2 @@
 node_modules
 coverage
-lib
```
Comment (alexander-akait) [inline .gitignore:3]: **NEED RETURN** before release (require for fast migration and testing)

**Comment by mightyaleksey** [comment]
@evilebottnawi hi, do you need help here?

Looks like if we drop the node@4 support, we can also drop babel and remove the lib folder totally.
I would like to suggest to include the postcss version update into the 4.0.0 release also.

**Comment by alexander-akait** [comment]
@sullenor sorry, don't have time on this right now, feel free to do what you plan, I will fix the problems that I solved here later, they are not critical

**Comment by alexander-akait** [comment]
Close in favor https://github.com/css-modules/icss-utils/pull/61

**Pull request closed without merging**